### ISSUE TITLE
'generate container' produces 'Unnecessary return' 

### ISSUE THREAD

**Issue by gihrig** [mxstbr/react-boilerplate#1231]
#After dependency update #1225 many new style errors are flagged.

To reproduce: 
1. Create a new container `npm run generate container`.
2. Accept all defaults for generator.
3. `npm run lint`.

Error
```
react-bp/app/containers/TestPage/sagas.js
5:3 error Unnecessary return statement no-useless-return
```

_Needs to be fixed in generator template <---_

return; -> return null; Or ???

Removing the `return` triggers `no-empty-fuction`

**Comment by gihrig** [comment]
Resolved in #1327.

Closing.

**Comment by gihrig** [comment]
A better solution. Handles the problems and guides the user (dev).

```javascript
// Individual exports for testing
export function* defaultSaga() {
  // See example in containers/HomePage/sagas.js
}
```
